### ISSUE TITLE
Application namespace

### ISSUE THREAD

**Issue by BobMali** [theweekendgeek/jambot#13]
I prefer using a real application namespace instead of the generic "App" which is with Laravel per default. Any suggestions for the Vendor/Package-name?

**Comment by theweekendgeek** [comment]
I agree.  i guess that jambot might already be sufficient as namespace?

**Comment by theweekendgeek** [comment]
I will make this part of the "getting ready"-milestone and I will assign this to you. Feel free to reassign if you disagree :)

**Comment by Ti-Jesu** [comment]
please i have a problem running the brunnergino/JamBot python project on my computer. i need someone to help me out
